README.md

```diff
@@ -18,8 +18,8 @@ Requirements
 ------------
 
 * Django 1.9 - 1.11
-* Python == 2.7
-* [edtf-validate](https://github.com/unt-libraries/edtf-validate) >= 1.0.0
+* Python == 3.7
+* [edtf-validate](https://github.com/unt-libraries/edtf-validate) >= 1.1.0
```

Comment:
Does this require specifically Python 3.7? We can list more than one version of Python if this supports more than one version.